etc/profile-m-z/vlc.profile

```diff
@@ -45,9 +47,9 @@ private-bin cvlc,nvlc,qvlc,rvlc,svlc,vlc
 private-dev
 private-tmp
 
-# dbus needed for MPRIS
-# dbus-user none
-# dbus-system none
-
-# mdwe is disabled due to breaking hardware accelerated decoding
-#memory-deny-write-execute
+dbus-user filter
+dbus-user.talk org.freedesktop.Notifications
+dbus-user.talk org.freedesktop.ScreenSaver
+dbus-user.talk org.mpris.MediaPlayer2.Player
+dbus-user.own org.mpris.MediaPlayer2.vlc
```

Comment:
nitpick: add this line above the 'dbus-user.talk <foo>' lines

Reply:
Will do, thanks!